lizaku/assignment3/assignment3.py

```diff
@@ -0,0 +1,127 @@
+class Dictionary:
+    '''
+    I implement the dictionary as a trie. As an input I give an array of words 
+    that are then transformed into a trie. isWord and isPrefix are methods of the Dictionary.
+    '''
+    def __init__(self, data):
+        self.data = self._make(data)
+    
+    def _make(self, words):
```

Comment:
Why did you separate this in a different method? Couldn't it be directly in the __init__()?

Reply:
I just have a stereotypic thinking that some major transformations shouldn't be in the __init__ function... Also, if I do several things with the data that I pass, it should be different functions, right?